compiler/test/dotty/tools/scripting/ScriptingTests.scala

```diff
@@ -146,6 +146,8 @@ class ScriptingTests:
     if touchedFile.exists then
       printf("success: executable jar created file %s\n", touchedFile)
     assert( touchedFile.exists, s"expected to find file ${touchedFile}" )
+    touchedFile.delete
+    assert( !touchedFile.exists, s"unable to delete ${touchedFile}" )
```

Comment:
I found that touchedFile.out was a leftover from this test. Now it should be deleted properly.